happi/cli.py

```diff
@@ -419,15 +417,17 @@ def transfer(ctx, name: str, target: str):
     # retrive client
     client = ctx.obj
     # verify name and target both exist and are valid
-    item = client.find_item(name=name)
+    try:
+        item = client.find_item(name=name)
+    except SearchError as e:
+        raise click.ClickException(str(e))
```

Comment:
What's the functional difference between raising specifically a `ClickException` vs some other conventional Python exception like `ValueError`?

I only ask because elsewhere in this file are two instances where we raise `RuntimeError` instead of `ClickException`

Reply:
`ClickException` is neatly printed back to the user (basically `Error: blah blah blah`, and it terminates the program), non-click exceptions dump a stacktrace. I'll look at the two other instances of `RuntimeError` in here to make sure I didn't miss anything.